GA-worksheet.md

```diff
@@ -7,18 +7,18 @@
 
 ##Project #2 Part #1: Header HTML
 1. What is the purpose of the header tag?
-
+The purpose of the header tag is to make sure the blog has an order in it's body and we have a heading inside the page. The header tag is another code for the body, but doesn't indent.
 2. What is a Navigation Bar?
+A Navigation Bar, helps us to link another page to our blog.
+3. What tag do we use to make a navigation bar? <   a   >
 
-3. What tag do we use to make a navigation bar? <      >
-
-4.  <ul> Stands for _____________________
-    <li> Stands for ______________________
+4.  <ul> Stands for _unwritten bullets____________________
```

Comment:
`<ul>` stands for unordered list, as opposed to ordered list. An unordered list has bullet points for each `<li>` list item.